bezpecnostni-doporuceni-open-source.md

```diff
@@ -541,3 +539,121 @@ Pokud jsou v jedné transakci měněny nebo přidávány hodnoty do více datab
 
 ### D.6 Databázové schéma je komentováno
 Tabulky a sloupce jsou vhodně komentovány, aby byl jasný jejich význam a vazby na jiné tabulky.
+
+
+## Dokumentace a architektonické znalosti
+
+Následující je platné vše vždy, pokud se jedná o informační systém veřejné správy, respektive nějakou komponentu či část, která je využívána jako ISVS a podléhá tedy Zákonu 365/2000 a povinnostem plynoucích z Národní architektury. Nejedná-li se o část ISVS, i tak se autor snaží maximální počet následujících principů splnit.
+
+Veškeré programové vybavení a programové potředky, jež veřejná správa využívá pro podporu výkonu veřejné správy jsou regulovány mimo jiné i legislativou k informačním systémům veřejné správy. Platí tedy, že pokud jsou součástí “informačních systémů veřejné správy” nebo “egulovaných informačních systémů! I části vytvořená jako otevřené dle tohoto doporučení, musí jejich rozvojem a implementací příslušný úřad plnit také povinnosti k ISVS a řízení informatiky (viz Zákon č. 365/2000 Sb., o informačních systémech veřejné správy a jeho prováděcí dokumenty). Je tedy vhodné, aby již součástí dokumentace otevřeného kódu systému či jeho části, byly také příslušné části architektonické dokumentace, aby se tak úřadům usnadnilo plnění povinností k ISVS.
```

Comment:
> jsou součástí “informačních systémů veřejné správy” nebo “egulovaných informačních systémů! I části vytvořená jako otevřené dle tohoto doporučení

Tady se zřejmě popletla písmenka s vykřičníky a uvozovkami. A možná by to vlastně měly být dvě různé věty. Také by bylo fajn používat české uvozovky, pokud jsou potřeba.